Warning: 100 points required, but 256 points sampled.
Warning: 10000 points required, but 25600 points sampled.
Compiling model...
'compile' took 2.906208 s

Training model...

Step      Train loss                                                      Test loss                                                       Test metric
0         [2.80e+00, 2.01e+00, 5.67e-01, 0.00e+00, 1.83e-01, 4.31e-02]    [2.02e+00, 2.01e+00, 5.67e-01, 0.00e+00, 1.83e-01, 4.31e-02]    []

Best model at step 0:
  train loss: 5.60e+00
  test loss: 4.82e+00
  test metric: []

'train' took 19.049097 s
/working_dir/src/pde.py:220: RuntimeWarning: divide by zero encountered in divide
  loss_weights = len(initial_losses) / initial_losses

Compiling model...
'compile' took 0.004373 s

Training model...

Step      Train loss                                                      Test loss                                                       Test metric
0         [6.00e+00, 6.00e+00, 6.00e+00, nan, 6.00e+00, 6.00e+00]         [4.34e+00, 6.00e+00, 6.00e+00, nan, 6.00e+00, 6.00e+00]         []

Best model at step 0:
  train loss: 5.60e+00
  test loss: 4.82e+00
  test metric: []

Epoch 1: saving model to ./tests/models/BioHeat_PINNs_date_time_20240711_135831/adam-1.pt ...

'train' took 24.492803 s

[2024-07-11 13:59:23,455][matplotlib.mathtext][INFO] - Substituting symbol L from STIXNonUnicode
[2024-07-11 13:59:23,523][matplotlib.mathtext][INFO] - Substituting symbol L from STIXNonUnicode
[2024-07-11 13:59:23,548][matplotlib.mathtext][INFO] - Substituting symbol L from STIXNonUnicode
[2024-07-11 13:59:23,573][matplotlib.mathtext][INFO] - Substituting symbol L from STIXNonUnicode
Error executing job with overrides: []
Traceback (most recent call last):
  File "/working_dir/./src/main.py", line 54, in main
    train.single_observer(prj, cfg.run, "0", cfg)
  File "/working_dir/src/train.py", line 110, in single_observer
    metrics = plots.plot_and_metrics(mo, n_test)
  File "/working_dir/src/plots.py", line 177, in plot_and_metrics
    g = gen_obsdata(n_test)
  File "/working_dir/src/plots.py", line 138, in gen_obsdata
    g = np.hstack((gen_testdata(n)))
  File "/opt/conda/lib/python3.10/site-packages/numpy/core/shape_base.py", line 359, in hstack
    return _nx.concatenate(arrs, 1, dtype=dtype, casting=casting)
ValueError: all the input array dimensions except for the concatenation axis must match exactly, but along dimension 0, the array at index 0 has size 2541 and the array at index 1 has size 5082

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
